package.json

```diff
@@ -11,9 +11,8 @@
     "lodash.chunk": "^4.2.0",
     "mssql": "^4.3.2",
     "n3": "^1.16.3",
-    "rdflib": "^2.2.21",
-    "request": "^2.88.0",
-    "request-promise": "^4.2.2"
+    "node-fetch": "^2.0.0",
```

Comment:
Latest version of `node-fetch` is v3.3.0 (see https://www.npmjs.com/package/node-fetch). Can we upgrade to that one?

Reply:
When v3 is used it causes errors because a dependent package called digest-fetch uses node-fetch by using require instead of import.

Reply:
I still get the following error :

2023-02-23T12:46:49.019354865Z Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /usr/src/build/node_modules/node-fetch/src/index.js
2023-02-23T12:46:49.019364856Z require() of ES modules is not supported.
2023-02-23T12:46:49.019373603Z require() of /usr/src/build/node_modules/node-fetch/src/index.js from /usr/src/build/app.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.